### DISCUSSION TITLE
SMTP configuration error.

### DISCUSSION THREAD

**Discussion by pravinrupnoor** [requarks/wiki#5335] [Error / Bug Report]
![image](https://user-images.githubusercontent.com/53506068/170423113-4328cec8-1895-4b27-827b-6a375ef20789.png)

error
### **140607810381696:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:** 

This is error coming while configuring smtp.
Please check.

**Comment by DarkArc** [comment]
Duplicate of https://github.com/requarks/wiki/discussions/3574
